wordpress-mobile/WordPress-iOS
# Issue/3844 Improves the appearance of the posts / pages list footer animation

Refs #3844.

I think ultimately the original issue is probably due to the estimated height for cells being different to the final height (just a guess), however these few tweaks improve the animation a lot. I've simply given the cells clear backgrounds and moved the footer view to the back. 

Before:

![footer-bug](https://cloud.githubusercontent.com/assets/4780/16683492/184eb640-44f8-11e6-8990-0d26f01e9012.gif)

After:

![footer-bug-2](https://cloud.githubusercontent.com/assets/4780/16683496/1f9ee65e-44f8-11e6-9828-87202dc9b797.gif)

### To test

I was able to replicate this simply by viewing the Posts list on a site with a single post, and the effect is easier to see with slow animations enabled. I guess also check you can't see any negative effects of making the post cell backgrounds clear.

Do you think this is an acceptable 'fix' for the original issue, or should we also look at preventing the animation in the first place?

Needs review: @aerych
